frontend/app/vault/controller.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller'
+import { computed } from '@ember/object'
+import { inject as service } from '@ember/service'
+
+export default Controller.extend({
+  router: service(),
+  detail: computed('router.currentRouteName', 'detail', function() {
```

Comment:
If that would actually work, it'd cause a loop.. You can't compute a property on itself!